docs/install/remote.md

```diff
@@ -91,6 +91,31 @@ mlrun.set_environment("http://localhost:8080", artifact_path="./")
 mlrun.set_environment("<remote-service-url>", access_key="xyz", username="joe")
 ```
 
+3. Run `mlrun config` command line to get (default) or set the environment variables into the default environment file. It will store all the configuration into the default environment file, without a need to edit your own environment file.
```

Comment:
3. Run `mlrun config` command line to get (default) or set the environment variables into the default environment file. It stores all of the configuration into the default environment file, and your own environment file does not need editing.

Reply:
Fixed